src/App.vue

```diff
@@ -74,7 +74,7 @@ export default {
       if (!this.userData)
         return "https://via.placeholder.com/256/2C2F33/FFFFFF?text=Loading";
       if (this.userData.avatar) {
-        return url + `avatars/${this.userData.id}/${this.userData.avatar}`;
+        return protocol + '://' + url + `avatars/${this.userData.id}/${this.userData.avatar}`;
```

Comment:
I have also added some general enhancements to make it easier to code. This changes the 'url' config to the 'url' and 'protocol' variables, to make it easier to connect things such as websockets